src/pages/react/why-react/index.md

```diff
@@ -15,5 +15,9 @@ The tools and software commonly utilized with React are incredibly well maintain
 ### Support
 React is created and maintained by the folks at Facebook and is used by individuals and companies all over the world in large volume. This means React is constantly improving and any problems you may have have probably already been asked on Stack Overflow.
 
+### Uses ES6
+Current version of react is designed to use es6 components where ever possible so it makes best use of new ES6 features such as 
+modules, generators ,iterators , arrow functions etc.
```

Comment:
Hi @geekyatri 

Thanks for opening this PR. Can you please edit this to use consistent spacing between the commas and the words? There is inconsistent spacing between every single one.

The correct spacing would be `word1, word2, word3, word4`.